packages/cli/src/util/__tests__/colorizer.spec.ts

```diff
@@ -0,0 +1,135 @@
+import chalk from 'chalk';
+import {
+  PRE_PARAM_VALUE_TAG,
+  POST_PARAM_VALUE_TAG,
+  attachTagsToParamsValues,
+  transformPathParamsValues
+} from '../colorizer';
+
+import { createPrismServerWithLogger } from '../createServer';
+import { PassThrough } from 'stream';
+import { createLogger } from '@stoplight/prism-core';
+import * as httpServer from '@stoplight/prism-http-server';
+
+const logStream = new PassThrough();
+const logInstance = createLogger('CLI', undefined, logStream);
+
+describe('colorizer', () => {
+
+  if (chalk.supportsColor) {
+    it('colorizes values for parameters', async () => {
+      // @ts-ignore
+      jest.spyOn(httpServer, 'createServer').mockImplementationOnce(() => {
+        return {
+          listen: () => {}
+        }
+      });
+
+      jest.spyOn(logInstance, 'note').mockImplementation((aString) => {
+        // @ts-ignore
+        const coloredParam = aString.match(/(endpoint\/)(.*)/)[2];
+
+        expect(coloredParam).toContain("\u001b[1m\u001b[36m");
+        expect(coloredParam).toContain("\u001b[39m\u001b[22m");
+      });
+
+      await createPrismServerWithLogger({
+        errors: false,
+        multiprocess: false,
+        host: 'localhost',
+        port: 9999,
+        cors: false,
+        document: './examples/test.yaml',
+        dynamic: false
+      }, logInstance)
+    })
+  }
```

Comment:
I would remove this test.

1. It has an `if` statement that is usually not ok for a test; in case the colours are not supported on the testing platform, this whole thing is dropped
2. It's doing weird things (ts-ignore, doing assertions in `mockImplementation` functions) which makes it hard to read and understand.

In general I feel the value of testing this compared to the cost of such test is too low to justify them.